index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html>
+   <head>
+      <title>Bimbala Desktop</title>
+      <!-- Include the Tailwind CSS stylesheet -->
+      <link href="assets/style.css" rel="stylesheet">
+   </head>
+   <body>
+      <section class="flex h-screen flex-col items-center md:flex-row">
+         <div class="hidden h-screen w-full bg-indigo-600 md:w-1/2 lg:block xl:w-2/3">
+            <img src="https://source.unsplash.com/random/?building" alt="" class="h-full w-full object-cover" />
+         </div>
+         <div
+            class="flex h-screen w-full items-center justify-center bg-white px-6 md:mx-auto md:w-1/2 md:max-w-md lg:max-w-full lg:px-16 xl:w-1/3 xl:px-12">
+            <div class="h-100 w-full">
+               <h1 class="mb-32 text-xl font-bold leading-tight md:text-2xl">Bimbala Desktop</h1>
+               <form id="nameForm" class="mt-6 space-y-4">
+                  <div>
+                     <label class="block text-gray-700">Enter name:</label>
+                     <input type="text" id="name" name="name"
+                        class="mt-2 w-full rounded-lg border bg-gray-200 px-4 py-3 focus:border-blue-500 focus:bg-white focus:outline-none"
+                        autofocus autocomplete required />
+                  </div>
+                  <button type="submit"
+                     class="mt-6 block w-full rounded-lg bg-indigo-500 px-4 py-3 font-semibold text-white hover:bg-indigo-400 focus:bg-indigo-400">Log
+                  In</button>
+               </form>
+               <hr class="my-6 w-full border-gray-300" />
+               <!-- Display the history of the last 5 entered names -->
+               <p class="mt-2">History:</p>
+               <ul id="nameHistory" class="pl-6 list-disc font-semibold text-blue-500"></ul>
+               <div class="h-100 w-full">
+                  <!-- Add a button to clean the history -->
+                  <button id="cleanHistory"
+                     class="mt-6 block w-full rounded-lg bg-red-500 px-4 py-3 font-semibold text-white hover:bg-red-400 focus:bg-red-400">
+                  Clean History
+                  </button>
```

Comment:
Let's change it to "Clear history" and change to color to be less dangerous-like.
```suggestion
                  <button id="cleanHistory"
                     class="mt-6 block w-full rounded-lg px-4 py-3 font-semibold text-black border-indigo-500  hover:bg-indigo-100 focus:bg-indigo-100">
                  Clear History
                  </button>
```

_Note: The css may need to be recompiled._